README.md

```diff
@@ -39,3 +39,32 @@ the prebuilt image or a customer collector build at this time.
 See the [recommended custom collector build
 configuration](./arrow/otelcolarrow-build.yaml) as a starting
 point.
+
+----
+### Contributing
```

Comment:
Let's actually move this to a new file called CONTRIBUTING.md as that matches how we handle it in other otel repos.